src/components/Input/NumericInput/NumericInput.tsx

```diff
@@ -125,6 +129,14 @@ export function NumericInput({
       max={max}
       step={step}
       hideControls={hideControls}
+      leftSection={
+        isOutsideRange &&
```

Comment:
I love this!   :D Looks great. 

Could we maybe add a `Tooltip` that says something like "The set value is outside of the min/max range"?  So that one can hover over the icon to see what it means?

It does add to an already existing issue of there not being enough space for Vector properties, but we should look at improving those anyways and since this is a feature that most people shouln't see, I don't think it's necessarily a problem

![image](https://github.com/user-attachments/assets/d96d8844-a080-4d0f-90d0-2fd8ded10745)